website/content/docs/developers/python/topologies.md

```diff
@@ -1,112 +1,253 @@
 ---
-title: Writing and Launching a Topology in Python
+title: Python Topologies
 ---
 
-Currently, support for developing a Heron topology in Python is still experimental.
-It is compatible with the Streamparse API, so Python topologies written
-for the Streamparse can be deployed on Heron with ease.
-This page describes how to write and launch a topology in Python, as well as
-how to convert a Streamparse topology to a PyHeron topology.
+Support for developing Heron topologies in Python is provided by a Python library called [`heronpy`](https://pypi.python.org/pypi/heronpy). Python topologies written using Storm's [streamparse](https://streamparse.readthedocs.io/en/latest/api.html) API are also compatible with Heron, which enables you to seamlessly [upgrade from Storm to Heron](/docs/upgrade-storm-to-heron) and keep your old topologies.
```

Comment:
Please remove the streamparse sentence since that is no longer supported. 
Also please change the "upgrade from Storm to Heron" to something along the lines written in architecture.md